scalameter-core/src/test/scala/org/scalameter/picklers/PicklersSpecification.scala

```diff
@@ -0,0 +1,89 @@
+package org.scalameter.picklers
+
+import java.util.Date
+import org.scalameter.picklers.Implicits._
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.{FunSuite, Matchers}
+
+
+class PicklerSpecification extends FunSuite with PropertyChecks with Matchers {
+  def validatePickler[T: Pickler](o: T) = implicitly[Pickler[T]].unpickle(implicitly[Pickler[T]].pickle(o)) should === (o)
+
+  test("Byte pickling") {
+    forAll { o: Byte =>
```

Comment:
Cool. Unfortunately, these look painfully like ScalaCheck's config params, and it seems that ScalaTest is wrapping ScalaCheck directly. And as far as I know, there are only ugly hacks to override RNG seeds in ScalaCheck :(

In this case, having random values is not paramount, but since it would be useful elsewhere, I had my hopes up for a second :)